File: beecrowd/python/bee1044.py
n1, n2 = map(int, input().split())

if n1 > n2 and n1 % n2 == 0:
    print('Sao Multiplos')
elif n2 > n1 and n2 % n1 == 0:
    print('Sao Multiplos')
else:
    print('Nao sao Multiplos')

File: javascript/bee1044.js
var [n1, n2] = require("fs").readFileSync("stdin", "utf8").split(" ").map(Number);

const resp = n1 % n2 === 0 || n2 % n1 === 0 ? "Sao Multiplos" : "Nao sao Multiplos";
console.log(resp);